lib/ansible/modules/cloud/amazon/ec2_vol_info.py

```diff
@@ -116,7 +117,10 @@ def list_ec2_volumes(connection, module, region):
     try:
         all_volumes = describe_volumes_with_backoff(connection, ansible_dict_to_boto3_filter_list(sanitized_filters))
     except ClientError as e:
-        module.fail_json(msg=e.response, exception=traceback.format_exc())
+        module.fail_json_aws(e, msg="Failed to describe volumes.")
+
+    # We add region to the volume info so we need it here
+    region = get_aws_connection_info(module)[0]
```

Comment:
Missed this - probably should be
```suggestion
    region = get_aws_connection_info(module, boto3=True)[0]
```